src/main/java/net/starlark/java/syntax/NodePrinter.java

```diff
@@ -14,6 +14,7 @@
 package net.starlark.java.syntax;
 
 import java.util.List;
+import net.starlark.java.syntax.Comprehension.Clause;
```

Comment:
Please don't import nested classes; always refer to Comprehension.Clause outside that class.